Makefile

```diff
@@ -51,7 +52,7 @@ all: Makefile .docker-image conf
 
 .PHONY: bash
 bash: .docker-image
-	$(call docker_run, /bin/bash)
+	$(call docker_run, sh -c "${SOURCE_OE}; exec /bin/bash")
```

Comment:
This for me with the following error:

$ make bash
docker run --privileged=true --device /dev/loop0:/dev/loop0 --device /dev/loop1:/dev/loop1 --device /dev/loop2:/dev/loop2 --device /dev/loop3:/dev/loop3 --device /dev/loop4:/dev/loop4 --device /dev/loop5:/dev/loop5 --device /dev/loop6:/dev/loop6 --device /dev/loop7:/dev/loop7 -it -v /home/ncosta/workspace/build-dev/poky:/home/ncosta/poky -v /home/ncosta/.ssh:/home/ncosta/.ssh -v /home/ncosta/workspace/build-dev/build-env/Makefile:/home/ncosta/Makefile -e EDITOR=vim -e POKY=/home/ncosta/poky -e TEMPLATECONF=/home/ncosta/poky/meta-pelion-edge/conf --rm ncosta/ww-build-env:latest  sh -c "source /home/ncosta/workspace/build-dev/poky/oe-init-build-env /home/ncosta/workspace/build-dev/poky/build; exec /bin/bash"
sh: /home/ncosta/workspace/build-dev/poky/oe-init-build-env: No such file or directory
Makefile:72: recipe for target 'bash' failed
make: *** [bash] Error 1

Reply:
Oh, huh. That's probably an issue. Let's put off this change until another PR. I'll rebase it out.